### ISSUE TITLE
Repeated Google tag ID

### ISSUE THREAD

**Issue by vincentmajor** [radity/raditian-free-hugo-theme#11]
The Google tag ID appears to be repeated twice and is not being recognized by Google Analytics.

Excerpt from the [demo site](https://themes.gohugo.io/theme/raditian-free-hugo-theme/) but the same happens with my local version:

```
<script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-000000GTM-000000"></script>
...
<script>
    (function (w, d, s, l, i) {
        w[l] = w[l] || [];
        w[l].push({
            'gtm.start': new Date().getTime(),
            event: 'gtm.js'
        });
        var f = d.getElementsByTagName(s)[0],
            j = d.createElement(s),
            dl = l != 'dataLayer' ? '&l=' + l : '';
        j.async = true;
        j.src =
            'https://www.googletagmanager.com/gtm.js?id=GTM-000000' + i + dl;
        f.parentNode.insertBefore(j, f);
    })(window, document, 'script', 'dataLayer', 'GTM-000000');
</script>
```
Note the repeated tag (`GTM-000000`) in the first line which appears to be caused by concatenating the tag to the url already containing the tag.

**Comment by akarshjg** [comment]
True, I fixed this issue locally by deleting GTM-000000 in j.src as shown below -

`<script>`
    (function (w, d, s, l, i) {
        w[l] = w[l] || [];
        w[l].push({
            'gtm.start': new Date().getTime(),
            event: 'gtm.js'
        });
        var f = d.getElementsByTagName(s)[0],
            j = d.createElement(s),
            dl = l != 'dataLayer' ? '&l=' + l : '';
        j.async = true;
        j.src =
            'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
        f.parentNode.insertBefore(j, f);
    })(window, document, 'script', 'dataLayer', '{{ .Site.Params.googleTagManagerID }}');
`</script>`

**Comment by ascalco** [comment]
Found the same issue. For future references, go inside "head.html", keep `{{ .Site.Params.googleTagManagerID }} `and remove `+i +dl`.

**Comment by kamui** [comment]
Ran into this as well. Thanks @ascalco!

Edit:

Actually, if you want to do this the way the GTM snippet is represented you want to change the line to:

```
            'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
```

Then, the script will inject the GTM code via the dataLayer.
